environments/staging/terraform.yml

```diff
@@ -173,9 +173,10 @@ proxy_servers:
 
 rds_instances:
   - identifier: "pg0-staging"
-    instance_type: "db.t2.medium"
+    instance_type: "db.t3.small"
     storage: 160
     multi_az: yes
+    engine_version: 9.6.9
```

Comment:
does `engine_version` == postgres version?

Reply:
Yeah, I adopted terraform's naming convention (which is maybe also AWS's?). It would be simple to change this to postgres version if you think that'd be better

Reply:
I don't have a preference on it especially since its a convention. It was fairly easy to realize what it should mean (I suppose its nice to have a generic name for "X_version" if we also run other managed services)